TramsDataApi.Test/Integration/TrustsIntegrationTests.cs

```diff
@@ -94,7 +97,7 @@ public async Task  ShouldReturnTrust_WhenSearchingByUkprn_AndTrustExists()
                         .TrustPerformanceAndRiskLinkToWorkplaceForEfficiencyIcfpReview,
                     NumberInTrust = testTrustData.NumberInTrust.ToString()
                 },
-                Academies = new List<AcademyResponse>(),
+                Academies = new List<EstablishmentResponse>(),
```

Comment:
I think we're safe to change this key & update the API contract as well, so it is consistent. Maybe needs a bit of thought since our consumers care about academies but can get anything

Reply:
Yeah I've avoided changing it up to this point, but as our endpoint doesn't currently filter on type it makes sense to return all establishments. If we find there's a need for retrieving a list of establishments by type, I think we'd be better off with an endpoint specifically for that purpose, probably?